packages/frontend-design-poc/src/components/PageLayout/PageLayout.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from './pageLayout.module.css';
+import React from 'react';
+
+interface LayoutProps {
+	header: React.ReactNode;
+	sidebar: React.ReactNode;
+	main: React.ReactNode;
+	footer: React.ReactNode;
+}
+
+export const PageLayout: React.FC<LayoutProps> = ({ header, sidebar, main, footer }) => {
+	return (
```

Comment:
I think I prefer the components themselves, e.g Footer, being composed as header, aside, main and footer intself. I would like to test the components in isolation  for wcag, not being depending on PageLayout.